.gitmodules

```diff
@@ -36,7 +36,7 @@ fxDONOTUSEurl = https://github.com/NGEET/fates
 [submodule "cism"]
 path = components/cism
 url = https://github.com/ESCOMP/CISM-wrapper
-fxtag = cismwrap_2_2_010
+fxtag = cismwrap_2_2_011
 fxrequired = ToplevelRequired
 # Standard Fork to compare to with "git fleximod test" to ensure personal forks aren't committed
 fxDONOTUSEurl = https://github.com/ESCOMP/CISM-wrapper
```

Comment:
Looking at ccs_config below. It's a branch tag and for ctsm5.4.000 we'll want to be a tag on main. I'm working on that part. We'll also then want it updated to the latest here. So that we get the updated needed for cism as well as the update we need for mksurfdata_esmnf for the ne3 grid.

Reply:
OK, this was tagged now, so this should be ccs_config1.0.66.

Reply:
If tests OK, might I add the changes from #3597 to this PR to simplify getting that one in?